.openshiftio/application.yaml

```diff
@@ -6,6 +6,14 @@ metadata:
     description: This template creates a Build Configuration using an S2I builder.
     tags: instant-app
 parameters:
+  - name: SUFFIX_NAME
+    description: The suffix name for the template objects
+    displayName: Suffix name
+    value: master
```

Comment:
does this have to be "master",  can we stick with latest?  i believe nodeshift uses latest and i would like to keep them somewhat in sync